lib/main.js

```diff
@@ -33,11 +41,19 @@ module.exports = {
   activate() {
     require('atom-package-deps').install('linter-jshint');
 
-    this.scopes = ['source.js', 'source.js-semantic'];
+    this.scopes = [];
```

Comment:
No need for this line (or the extra two blank lines), observing the value below will assign an initial value.